twitter/test_selenium_twitter_scraper.py

```diff
@@ -195,12 +184,16 @@ def store_tweet_data_payload(searchQuery, dataPayload):
 
     logger.info("<< STORED Tweets Payload >>")
 
-    # TODO: Removes created file to save space
+    #-- Send collected Twitter data payload to RabbitMQ Queue
+    # run_sync_script_rabbitmq((searchQuery, dataPayload))
+    publish_stock(binaryFile,'','scraped_data', logger) #TODO: Check if binaryfile payload sent to RabbitmQ can be converted back with right data
```

Comment:
Did you verify if the data is actually getting pushed into the queue? Or is that part of what you mean to do in the TODO item

Reply:
No. Will get onto it